GoogleForCreators/web-stories-wp
# Adapt popup height to contents

## Summary

Makes the popup component dynamically adapt to height changes of its content, thereby preventing scrollable overflow (especially visible on the calendar element which switching views, see https://github.com/google/web-stories-wp/pull/1743#discussion_r427682509). 

## To-do

- [x] Trigger the height change automatically without need for a scroll event

## User-facing changes

- Link popups are now fixed (no longer showing an unnecessary scroll bar)
- Calendar will now be fully visible without need for scrolling

## Testing Instructions
I
1. Open the calendar.
2. Scroll up (so that the calendar would be in the bottom of the viewport, resize the window if needed to achieve that)
3. Now click on the month (May 2020) to open the month view - this will change the content height.
4. Verify that the calendar/popup adjusts the size and all parts of the calendar content are still visible.

II
1. Add a video element.
2. Resize it to be very narrow.
3. Now hover over it: verify the Play button is visible outside (just aside) of the Video element.
---



Fixes #1931 , see context on https://github.com/google/web-stories-wp/pull/1743#discussion_r427682509
Part of #1082
Fixes #2343
